src/components/DepartmentsList.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { getDepartments } from '../api/api';
+import Department from './Department';
+import Loader from './Loader';
+
+export default class DepartmentsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      departments: [],
+      isLoading: false,
+      error: null,
+    };
+  }
```

Comment:
You also may not use the constructor here and use state as property
```suggestion
     state = {...}
```